packages/calcite-components/src/custom-theme.stories.ts

```diff
@@ -104,7 +105,9 @@ const kitchenSink = (args: Record<string, string>, useTestValues = false) =>
           <div>${checkbox}</div>
           ${chips} ${pagination} ${slider}
         </div>
-        <div class="demo-column">${datePicker} ${tabs} ${loader} ${calciteSwitch}</div>
+        <div class="demo-column">${datePicker} ${tabs} ${loader} ${calciteSwitch}
+        <div style="margin-top: 40px">${progress}</div>
```

Comment:
To reduce usage of inline styles, can you update the switch story to use a `calcite-label` (with `layout="inline"`)?